docs/usage/cli/filter.rst

```diff
@@ -35,7 +43,7 @@ These can be specified in a separate file:
     COL/FLR_00034/2015
     ...
 
-To drop such strains, you can pass the name of this file to the augur filter command:
+To drop such strains, you can use ``--exclude``:
```

Comment:
Maybe we should somehow explicitly connect the example outlier code above with the `config/dropped_strains.txt` filename used below?

Reply:
Done: af4d6274c2eb9866a4e471e8d7748588b692fcaf

I chose the name `exclude.txt` based on discussion elsewhere: https://github.com/nextstrain/dengue/pull/26#discussion_r1486780307